pkg/controller/kubelet-config/helpers.go

```diff
@@ -33,6 +33,9 @@ const (
 	managedFeaturesKeyPrefix      = "98"
 	managedKubeletConfigKeyPrefix = "99"
 	protectKernelDefaultsStr      = "\"protectKernelDefaults\":false"
+	cgroupsV1DeprecationMsg       = "CGroupsv1 support will be deprecated soon by Openshift, consider switching to CGroupsV2 mode"
```

Comment:
Just to clarify this, the message seems to imply that we're not deprecating CGroupsV1 until a later release. Is that correct? Should we instead say that CGroupsV1 **is** deprecated, and the user is strongly recommended to switch to CGroupsV2? Is there any plan to remove support for CGroupsV1?

Reply:
It seems that the CGroupV1 deprecation message has to be part of OCP-4.18 and the support for the same would be removed in OCP-4.19(RHEL-10) as the RHEL-10 would not support it
cc: @harche